modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -71,61 +70,33 @@ describe('test tss helper functions', function () {
   describe('encryptNShare', function () {
     it('should encrypt n share', async function () {
       for (let i = 2; i <= 3; i++) {
-        const encryptedNShare = await ECDSAMethods.encryptNShare({
-          keyShare: userKeyShare,
-          recipientIndex: i,
-          senderGpgPrivateArmor: userGpgKeypair.privateKey,
-          recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-        });
-        const decryptedMessage = await readSignedMessage(encryptedNShare.encryptedPrivateShare, userGpgKeypair.publicKey, bitgoGpgKeypair.privateKey);
-        decryptedMessage.should.equal(userKeyShare.nShares[i].u);
-        const publicKey = userKeyShare.pShare.y;
+        const encryptedNShare = await ECDSAMethods.encryptNShare(userKeyShare, i, bitgoGpgKeypair.publicKey, userGpgKeypair.privateKey);
+        const decryptedMessage = '0x' + await readSignedMessage(encryptedNShare.encryptedPrivateShare, userGpgKeypair.publicKey, bitgoGpgKeypair.privateKey);
+        decryptedMessage.should.equal(userKeyShare.nShares[i].u + userKeyShare.nShares[i].n.slice(2));
+        const publicKey = userKeyShare.pShare.y + userKeyShare.pShare.chaincode;
         encryptedNShare.i.should.equal(i);
         encryptedNShare.j.should.equal(1);
-        encryptedNShare.publicShare.should.equal(publicKey);
+        ('0x' + encryptedNShare.publicShare).should.equal(publicKey);
       }
     });
 
     it('should error for invalid recipient', async function () {
-      await encryptNShare({
-        keyShare: userKeyShare,
-        recipientIndex: 1,
-        senderGpgPrivateArmor: userGpgKeypair.privateKey,
-        recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-      }).should.be.rejectedWith('Invalid recipient');
-      await encryptNShare({
-        keyShare: backupKeyShare,
-        recipientIndex: 2,
-        senderGpgPrivateArmor: userGpgKeypair.privateKey,
-        recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-      }).should.be.rejectedWith('Invalid recipient');
-      await encryptNShare({
-        keyShare: bitgoKeyShare,
-        recipientIndex: 3,
-        senderGpgPrivateArmor: userGpgKeypair.privateKey,
-        recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-      }).should.be.rejectedWith('Invalid recipient');
+      await encryptNShare(userKeyShare, 1, userGpgKeypair.privateKey, bitgoGpgKeypair.publicKey).should.be.rejectedWith('Invalid recipient');
+      await encryptNShare(backupKeyShare, 2, userGpgKeypair.privateKey, bitgoGpgKeypair.publicKey ).should.be.rejectedWith('Invalid recipient');
+      await encryptNShare(bitgoKeyShare, 3, userGpgKeypair.privateKey, bitgoGpgKeypair.publicKey ).should.be.rejectedWith('Invalid recipient');
     });
   });
 
   describe('createCombinedKey', function () {
     it('should create combined user key', async function () {
```

Comment:
Since logic for the combine user key and backup key is similar, lets use a config and loop it to test to resuse the same logic, something like I have done [here](https://github.com/BitGo/BitGoJS/blob/df5768f1ef46af117add35f3fefd6adf125d87a3/modules/account-lib/test/unit/mpc/tss/ecdsa/ecdsa.ts#L102)